app/tests/test_date_formatter.py

```diff
@@ -0,0 +1,38 @@
+from app.main.util.date_formatter import get_date_range
+
+
+def test_browse_data_date_from_filter_exception_raised(app, caplog):
+    """
+    Given a date_from value error
```

Comment:
```suggestion
    Given an invalid date_from value
```

Reply:
changes done as requested and committed to branch , available for review

Reply:
done the changes as requested